### PR TITLE
Table component jose

### PR DESCRIPTION
**Pull request by jcrobles1989**
### Description

**What work was done?**

A reusable table component was created, with delete and edit functionality (edit is still being worked on).

**Why was this work done?**

So that data could be rendered in a tabular format and the component used throughout the application. 

**What feature / user story is it for?**
(Link if possible) 

**Any relevant links or images / screenshots**

### Type of change
Please delete options that are not relevant.
[x] New feature (non-breaking change which adds functionality)

Change Status
[] Completed, ready to review and merge
Has This Been Tested
[] Yes

### Checklist
[] My code follows the style guidelines of this project
[x] I have performed a self-review of my own code
[x] My code has been reviewed by at least one peer
[] I have commented my code, particularly in hard-to-understand areas
[x] I have made corresponding changes to the documentation
[] My changes generate no new warnings
[] There are no merge conflicts

### CODE REVIEW

File: src/components/common/Table/Table.js [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+    name: 'Bilbo Baggins',
+    role: 'Program Manager',
+    programs: ['Prevention', 'Aftercare', 'Sheltering'],
+  },
+  {
+    name: 'Frodo Baggins',
+    role: 'Service Provider',
+    programs: ['Sheltering', 'Aftercare'],
+  },
+];
```
Comment (princeali415) [inline src/components/common/Table/Table.js:77]: The user data will be fetched from backend in future but for first run this is good

File: src/styles/components/_table.less [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    margin-bottom: 8px;
+    margin-left: 8px;
+  }
+
+  .editable-row .ant-form-item-explain {
+    position: absolute;
+    top: 100%;
+    font-size: 12px;
+  }
+}
```
Comment (RubesWorld) [inline src/styles/components/_table.less:30]: Can we convert this to Sass instead of Less?

**Review by princeali415 — CHANGES_REQUESTED** [review]
Table is good, lets delete the employee page u created and import the table to the main employee page the team has been using so far. We are also using sass not less for styling so talk to ruben about that and he'll help you convert. Overall good work on the table!!

File: src/index.js [hunk 3]
```diff
@@ -25,7 +25,6 @@ import { Dashboard } from './components/pages/Dashboard';
 import { LandingPage } from './components/pages/Landing';
 import { config } from './utils/oktaConfig';
 import { LoadingComponent } from './components/common';
-import { EmployeesPage } from './components/pages/AdminPages/index';
 
 import './styles/styles.less';
```
Comment (princeali415) [inline src/index.js:29]: still need to remove this import

**Review by princeali415 — APPROVED** [review]
Still need to remove .less imports but will merge it now so we can work with the tables

**Pull request merged**